lib/bokken_web/router.ex

```diff
@@ -19,6 +19,14 @@ defmodule BokkenWeb.Router do
     plug BokkenWeb.Auth.AllowedRoles, [:organizer]
   end
 
+  pipeline :auth do
```

Comment:
I think something like `jwt_auth` would be better name :thinking:

Reply:
That's actually more clear, I'll change it.